docs/troubleshooting/runtime.md

```diff
@@ -207,3 +207,15 @@ class MyApp {
   }
 }
 ```
+
+<!-- This is referenced in Ionic Framework component documentation so we explicitly define the anchor so it remains consistent. -->
+
+## Accessing `this` in a function callback returns `undefined` {#accessing-this}
+
+Certain components, such as [counterFormatter on ion-input](../api/input#counterformatter) and [pinFormatter on ion-range](../api/input#pinformatter), allow developers to pass callbacks. It's important that developers bind the correct `this` value if they plan to access `this` from within the context of the callback. Developers may need to access `this` when using Angular components or when using class components in React. There are two ways to bind `this`:
+
+The first way to bind `this` is to use the `bind()` method on a function instance. If a developer wanted to pass a callback called `counterFormatterFn`, then they would write `counterFormatterFn.bind(this)`.
```

Comment:
```suggestion
Certain components, such as [counterFormatter on ion-input](../api/input#counterformatter) and [pinFormatter on ion-range](../api/input#pinformatter), allow developers to pass callbacks. It's important that you bind the correct `this` value if you plan to access `this` from within the context of the callback. You may need to access `this` when using Angular components or when using class components in React. There are two ways to bind `this`:

The first way to bind `this` is to use the `bind()` method on a function instance. If you want to pass a callback called `counterFormatterFn`, then you would write `counterFormatterFn.bind(this)`.
```
Optional, but the rest of this page uses second person, so this makes it more consistent.